src/extension.ts

```diff
@@ -175,6 +178,9 @@ export function activate(context: vscode.ExtensionContext) {
       return documentation;
     }
   }
+  function escapeTabStopSign(value){
```

Comment:
Why is the $ sign special? Are there other characters that are special and that need t be escaped too?

Reply:
$ defines interpolation expression, there are other special characters like {} but they are relevant only in context of $ sign, e.g. : 

- ${expression} - special,  interpolation  that evaluates
- $1 - special, tab stop 
- {} - not special, not evaluates